__tests__/api/api.test.js

```diff
@@ -166,6 +166,61 @@ describe("GET /api/articles", () => {
   });
 });
 
+describe("GET /api/articles/:article_id", () => {
+  test("200: responds with a status of 200 and an object representing article_id 1", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        expect(comments).toBeInstanceOf(Array);
+        expect(comments).toHaveLength(11);
+        comments.forEach((comment) => {
+          expect(comment).toHaveProperty("comment_id", expect.any(Number));
+          expect(comment).toHaveProperty("votes", expect.any(Number));
+          expect(comment).toHaveProperty("created_at", expect.any(String));
+          expect(comment).toHaveProperty("author", expect.any(String));
+          expect(comment).toHaveProperty("body", expect.any(String));
+          expect(comment).toHaveProperty("article_id", expect.any(Number));
+          expect(comments[0]).toMatchObject({
+            comment_id: 5,
+            body: "I hate streaming noses",
+            votes: 0,
+            author: "icellusedkars",
+            article_id: 1,
+            created_at: "2020-11-03T21:00:00.000Z",
+          });
+        });
+      });
+  });
+  test("Comments should be sorted by most recent comments first.", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .then(({ body }) => {
+        const { comments } = body;
+        expect(comments).toBeSortedBy('created_at', {descending: true});
+      });
+  });
```

Comment:
Another 200 status test to check if the id exits but the array is empty is needed here. See my comments in the models that relates to this 👀